scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -707,6 +708,10 @@ def __start_docker_logs( self, docker_logs ):
                     global_log.info( "Error adding log '%s' to log watcher - %s" % (log['log_config']['path'], e) )
 
             if self._use_raw_logs:
+                log_path = log['log_config']['path']
+                if not os.path.exists(log_path):
+                    global_log.warn("Missing file detected for container log path '%s'. Please ensure that "
+                                    "/var/lib/docker/containers (or custom storage directory) is mounted." % log_path)
```

Comment:
'custom storage directory' is not so clear.  I would reword the second sentence to something like:

"Please ensure that the host's Docker container directory (by default `/var/lib/docker/containers`) has been mounted in the Scalyr Agent container."